cluster-operator/src/main/java/io/strimzi/operator/cluster/model/metrics/StrimziMetricsReporterModel.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model.metrics;
+
+import io.strimzi.api.kafka.model.common.HasConfigurableMetrics;
+import io.strimzi.api.kafka.model.common.metrics.StrimziMetricsReporter;
+import io.strimzi.operator.common.model.InvalidResourceException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+import java.util.regex.PatternSyntaxException;
+
+/**
+ * Model for the Strimzi Metrics Reporter Kafka plugin.
+ */
+public class StrimziMetricsReporterModel implements MetricsModel {
+    /**
+     * Fully qualified class name of the Strimzi Kafka Prometheus Metrics Reporter.
+     */
+    private final boolean isEnabled;
+    private final List<String> allowList;
+
+    /**
+     * Constructs the Metrics Model for managing configurable metrics to Strimzi.
+     *
+     * @param spec Custom resource section configuring metrics.
+     */
+    public StrimziMetricsReporterModel(HasConfigurableMetrics spec) {
```

Comment:
Do we really need this constructor? Just pass the dummy value from where are you calling it. It seems to be used in tests only anyway.

Reply:
The constructor has been removed and the tests refactored to allow for the removal.